mono/sgen/sgen-gc.h

```diff
@@ -127,16 +127,40 @@ extern guint64 stat_objects_copied_major;
 		g_error (__VA_ARGS__);	\
 } } while (0)
 
+#ifndef HOST_WIN32
+# define LOG_TIMESTAMP  \
+	do {	\
+		time_t t;									\
+		struct tm tod;									\
+		time(&t);									\
+		localtime_r(&t, &tod);								\
+		strftime(logTime, sizeof(logTime), "%F %T", &tod);				\
+	} while (0)
+#else
+# define LOG_TIMESTAMP  \
+	do {	\
+		time_t t;									\
+		struct tm *tod;									\
+		time(&t);									\
+		tod = localtime(&t);								\
+		strftime(logTime, sizeof(logTime), "%F %T", tod);				\
```

Comment:
The formatting strings %F %T are not supported on WIndows accoring to MSDN:
https://msdn.microsoft.com/en-us/library/fe06s4ak.aspx?f=255&MSPPError=-2147217396

but a quick test on VS 2015 c-runtime gave expected results using both %F and %T so I believe this will work as expected on windows as well.